package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nahmii-sdk",
-  "version": "1.0.0-beta.13",
+  "version": "2.0.0-beta.1",
```

Comment:
This will still be v1.0.0. Until we have an official 1.0 out we dont bump the primary versions, only the pre-release beta number.

Reply:
:+1:

Reply:
Need another bump after merging in latest "master"